code_schemes/waste_disposal_improvement.json

```diff
@@ -0,0 +1,86 @@
+{
+    "Version": "0.0.0.1",
+    "Name": "Waste disposal improvement?",
+    "SchemeID": "Scheme-4e566f117e25",
+    "Codes": [{
+        "VisibleInCoda": true,
+        "NumericValue": 0,
+        "CodeID": "code-72db9807",
+        "CodeType": "Normal",
+        "StringValue": "increase Pickup times",
```

Comment:
Our convention here so far has been for string values to be lower case and underscore separated e.g. "increase_pickup_times"

Reply:
@as2388  thanks will make the changes.